pages/depthChart.py

```diff
@@ -6,13 +6,13 @@
 from immaculateGridCalculations.complexFormulas import get_war, get_grouped_fielding
```

Comment:
I had to do some copy pasting because I had my files all sorts of different from what y'all had. That would be my guess that it's changes in white space because as far as I'm aware the only change I really made was doing a copy paste from main